numba/inline_closurecall.py

```diff
@@ -371,7 +371,7 @@ def inline_closure_call(func_ir, glbls, block, i, callee, typingctx=None,
     if work_list != None:
         for block in new_blocks:
             work_list.append(block)
-    return callee_blocks
+    return callee_blocks, var_dict
```

Comment:
This change (and what is currently returned) ought to be added to the docstring for `inline_closure_call`.

Reply:
done